insights/combiners/tests/test_lvm.py

```diff
@@ -452,3 +453,169 @@ def test_combiner_lvs(lvm_data):
             for lv in filtered_lvm_info:
                 assert lv.LV in ['pool7', '[pool7_tdata]', '[pool7_tmeta]']
             assert Lvm.LvVgName(LV='pool7', VG='data7') in filtered_lvm_info
+
+
+@pytest.fixture
+def lvm_data_all():
+    local = {}
+
+    vgsall = VgsAll(context_wrap(VGS_NO_HEADINGS))
+    pvsall = PvsAll(context_wrap(PVS_NO_HEADINGS))
+    lvsall = LvsAll(context_wrap(LVS_NO_HEADINGS))
+    shared_list = [
+        {VgsAll: vgsall},
+        {PvsAll: pvsall},
+        {LvsAll: lvsall},
+    ]
+    LvmData = namedtuple('LvmData', ['lvm_info', 'shared'])
+    yield [LvmData(LvmAll(local, shared), shared) for shared in shared_list]
+
+
+def test_combiner_vgsall(lvm_data_all):
+    has_vgsall = False
+    for data in lvm_data_all:
+        if VgsAll in data.shared:
+            has_vgsall = True
+            lvm_info = data.lvm_info
+            assert lvm_info.volume_groups is not None
+            assert len(list(lvm_info.volume_groups)) == 8
+            assert lvm_info.volume_groups['data7']['VG_UUID'] == 'r58iLs-sYla-lC4h-UURW-bZOG-P2Yd-lambpM'
+            assert lvm_info.volume_group_names == {
+                'data1', 'data2', 'data3', 'data4', 'data5', 'data6', 'data7',
+                'rhel_ceehadoop1'
+            }
+            if VgsAll in data.shared:
```

Comment:
This `if` can be removed and it's `else` can be shifted out as the `else` against to the `if` at line `477`.